db/seeds.d/103-provisioning_templates.rb

```diff
@@ -9,20 +9,25 @@
 # Provisioning Templates
 
 kinds = [:provision, :finish, :user_data].inject({}) do |hash, kind|
-  hash[kind] = TemplateKind.find_by_name(kind)
+  hash[kind] = TemplateKind.find_by(:name => kind)
   hash
 end
 
-defaults = {:vendor => "Katello", :default => true, :locked => true}
-
 templates = [{:name => "Katello Kickstart Default",           :source => "kickstart-katello.erb",      :template_kind => kinds[:provision]},
              {:name => "Katello Kickstart Default User Data", :source => "userdata-katello.erb",       :template_kind => kinds[:user_data]},
              {:name => "Katello Kickstart Default Finish",    :source => "finish-katello.erb",         :template_kind => kinds[:finish]},
              {:name => "subscription_manager_registration",   :source => "snippets/_subscription_manager_registration.erb", :snippet => true}]
 
 templates.each do |template|
-  template[:template] = File.read(File.join(Katello::Engine.root, "app/views/foreman/unattended", template.delete(:source)))
```

Comment:
That may have been removed by accident, I'll see what happens when I add it back in